tests/bchar_test.ml

```diff
@@ -4,108 +4,108 @@ open Bchar
 
 let suite = 
   describe "Bchar" (fun () -> [
-    test "lowerCaseIfAscii - 'A'" (fun () ->
-      'A' |> lowerCaseIfAscii
-          |> Expect.toEqual 'a');
-    test "lowerCaseIfAscii - '0'" (fun () ->
-      '0' |> lowerCaseIfAscii
-          |> Expect.toEqual '0');
+  test "lowerCaseIfAscii - 'A'" (fun () ->
+    'A' |> lowerCaseIfAscii
+        |> Expect.toEqual 'a');
+  test "lowerCaseIfAscii - '0'" (fun () ->
+    '0' |> lowerCaseIfAscii
+        |> Expect.toEqual '0');
 
-    test "fromInt - 48" (fun () ->
-      48 |> fromInt
-         |> Expect.toEqual (Some '0'));
-    test "fromInt - 256" (fun () ->
-      256 |> fromInt
-          |> Expect.toEqual None);
+  test "fromInt - 48" (fun () ->
+    48 |> fromInt
+       |> Expect.toEqual (Some '0'));
+  test "fromInt - 256" (fun () ->
+    256 |> fromInt
+        |> Expect.toEqual None);
 
-    test "fromIntOrRaise - 48" (fun () ->
-      48 |> fromIntOrRaise
-         |> Expect.toEqual '0');
-    test "fromIntOrRaise - 256" (fun () ->
-      (fun () ->
-        256 |> fromIntOrRaise)
-            |> Expect.toRaise);
-    test "fromIntOrRaise - (~-1)" (fun () ->
-      (fun () ->
-        (~-1) |> fromIntOrRaise)
-              |> Expect.toRaise);
+  test "fromIntOrRaise - 48" (fun () ->
+    48 |> fromIntOrRaise
+       |> Expect.toEqual '0');
+  test "fromIntOrRaise - 256" (fun () ->
+    (fun () ->
+    256 |> fromIntOrRaise)
```

Comment:
This (and the next test) should have the body of the lambda indented one more level